tests/e2e/specs/BoundarySignalEvent.spec.js

```diff
@@ -0,0 +1,54 @@
+import {
+  dragFromSourceToDest,
+  getElementAtPosition,
+  moveElement,
+  removeIndentationAndLinebreaks,
+  setBoundaryEvent,
+  typeIntoTextInput,
+  waitToRenderAllShapes,
+} from '../support/utils';
+import { nodeTypes } from '../support/constants';
+
+describe('Boundary Signal Event', () => {
+  const taskPosition = { x: 200, y: 200 };
+  const boundarySignalEventPosition = { x: 260, y: 200 };
+
+  beforeEach(() => {
+    dragFromSourceToDest(nodeTypes.task, taskPosition);
+    setBoundaryEvent(nodeTypes.boundarySignalEvent, taskPosition);
+  });
+
+  it('update boundary signal event properties element', () => {
+    moveElement(taskPosition, boundarySignalEventPosition.x, boundarySignalEventPosition.y);
```

Comment:
What's the reason for moving the shape like this? It doesn't look like you make any assertions against the task's position.

Reply:
copy pasta